extensions/ql-vscode/src/pure/time.ts

```diff
@@ -43,20 +44,23 @@ export function humanizeRelativeTime(relativeTimeMillis?: number) {
 
 /**
  * Converts a number of milliseconds into a human-readable string with units, indicating an amount of time.
- * Negative numbers have no meaning and are considered to be "Less than a minute".
+ * Negative numbers have no meaning and are considered to be "Less than a second".
  *
  * @param millis The number of milliseconds to convert.
- * @returns A humanized duration. For example, "2 minutes", "2 hours", "2 days", or "2 months".
+ * @returns A humanized duration. For example, "2 seconds", "2 minutes", "2 hours", "2 days", or "2 months".
  */
 export function humanizeUnit(millis?: number): string {
   // assume a blank or empty string is a zero
   // assume anything less than 0 is a zero
-  if (!millis || millis < ONE_MINUTE_IN_MS) {
-    return 'Less than a minute';
+  if (!millis || millis < ONE_SECOND_IN_MS) {
```

Comment:
No, I haven't. I figured this would be fine since it's only used in 1 other place right now: the panel for a remote query. We are essentially recreating that panel with a better design and support for live results, so I don't think it matters that the duration for queries to change from less than 1 minute to 12 seconds. This will also only happen for failed or cancelled queries since other queries will almost always take more than 1 minute to complete.

Reply:
I think we should aim to flag anything that changes existing behaviour with Product/UX. I expect it'll be okay (and I agree with you that the change makes sense), but I want to make sure we don't surprise anyone :)